src/harness/scripts/migrate-values-0.9.x.sh

```diff
@@ -58,6 +58,17 @@ else
   exit 1
 fi
 
+required_yq_version="4.35.0"
+installed_yq_version=$(yq --version | cut -d ' ' -f 4 | sed 's/v//')
+
+# Compare versions
+if [[ "$(printf '%s\n' "$required_yq_version" "$installed_yq_version" | sort -V | head -n1)" != "$required_yq_version" ]]; then
```

Comment:
what if customer has higher version than required version ?

Reply:
it will work, its doing a sort and comparing the smaller value between required_yq_version and installed_yq_version and checking with required_yq_version. If it matches, it means, installed_yq_version > required_yq_version which proceeds further to the script

Reply:
Got it